src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,31 +26,19 @@ public void run() {
         }).start();
     }
 
+    public static int sumAll(List<Integer> numbers, Conditional c) {
+        return numbers.stream().filter(number -> c.test(number)).mapToInt(number -> number.intValue()).sum();
```

Comment:
```suggestion
        return numbers.stream()
        .filter(number -> c.test(number))
        .mapToInt(number -> number.intValue())
        .sum();
```
스트림은 기능단위로 줄바꿈하여 사용하는걸 권장드립니다 :)